cves/kernel/CVE-2014-9730.yml

```diff
@@ -75,7 +86,7 @@ bugs_instructions: |
     * Mentioned in mailing list discussions
     * References from NVD entry
     * Various other places
-bugs: []
+bugs: [1228229]
```

Comment:
should be a link to https://bugzilla.redhat.com/show_bug.cgi?id=1228229 rather than just the number